app/src/server/utils/generateApiKey.ts

```diff
@@ -0,0 +1,11 @@
+const KEY_LENGTH = 42;
+
+export const generateApiKey = () => {
+  const chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
+  let randomChars = "";
+  for (let i = 0; i < KEY_LENGTH; i++) {
+    randomChars += chars.charAt(Math.floor(Math.random() * chars.length));
```

Comment:
Can we use a dedicated lib for this? There have got to be a bunch and `Math.random` isn't actually cryptographically secure.

Reply:
We are now using `crypto-random-string`